docs/language/syntax.rst

```diff
@@ -175,6 +177,11 @@ Python-legal names. The steps are as follows:
    Thus, ``green☘`` becomes ``hyx_greenXshamrockX``, ``if`` becomes
    ``hyx_if``, and ``is_-_has_dashes`` becomes ``hyx_is_XhyphenHminusX_has_dashes``.
 
+.. note::
+
+   A symbol of a reserved word (such as ``if``) will not be made legal if used as
```

Comment:
Hmm, I think the other new docs should be sufficient, putting a note here in the mangling section is more confusing than not